packages/tenant-process/test/testAddCertifiedAttribute.ts

```diff
@@ -0,0 +1,256 @@
+import { fail } from "assert";
+import { attributeKind } from "pagopa-interop-models";
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import { readLastEventByStreamId } from "pagopa-interop-commons-test/index.js";
+import {
+  generateId,
+  Tenant,
+  Attribute,
+  unsafeBrandId,
+  protobufDecoder,
+  TenantCertifiedAttributeAssignedV2,
+  fromTenantKindV2,
+  toTenantV2,
+} from "pagopa-interop-models";
+import { describe, it, expect, vi } from "vitest";
+import {
+  tenantNotFound,
+  attributeNotFound,
+  tenantIsNotACertifier,
+  certifiedAttributeOriginIsNotCompliantWithCertifier,
+  certifiedAttributeAlreadyAssigned,
+} from "../src/model/domain/errors.js";
+import { ApiCertifiedTenantAttributeSeed } from "../src/model/types.js";
+import {
+  currentDate,
+  getMockAuthData,
+  addOneAttribute,
+  addOneTenant,
+  getMockTenant,
+  getMockCertifiedTenantAttribute,
+} from "./utils.js";
+import {
+  postgresDB,
+  attributes,
+  tenants,
+  tenantService,
+} from "./tenant.integration.test.js";
+
+export const testAddCertifiedAttribute = (): ReturnType<typeof describe> =>
+  describe("addCertifiedAttribute", async () => {
```

Comment:
Starting now let's use follow [this example](https://github.com/pagopa/interop-be-monorepo/pull/439/files#diff-35d2f3828797d06c66f2953b16a924c9bc9d2b6c8a880af5297e290d45ab434aR53) for creating tests.

Move everything you declare here inside a `beforeAll`, including the fakeTimers and in the `afterAll` you call the realTimers, and then remove them from the various tests since they will already by included with this change